src/CreditCardForm.js

```diff
@@ -218,6 +221,7 @@ export default class CreditCardForm extends Component {
       createAccount = false,
       inputStyles,
       overrideProps = {},
+      postalCodeValidations = null,
```

Comment:
Why null versus default to "text"? It's highly unlikely people want a US-only credit card form.